### PR TITLE
Feat add ledgers

### PR DESCRIPTION
**Pull request by mf222**
Se añaden los modelos de LedgerAccount, LedgerLine, Wallet y Accountable.

### CODE REVIEW

File: app/models/ledger_account.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
+class LedgerAccount < ApplicationRecord
+  belongs_to :accountable, polymorphic: true
+  has_many :ledger_lines
```
Comment (monkeyci) [inline app/models/ledger_account.rb]: Specify a :dependent option.

File: app/models/concerns/accountable.rb [hunk 2]
```diff
@@ -0,0 +1,8 @@
+
+module Accountable
+  extend ActiveSupport::Concern
+
+  included do
+    has_many :ledger_lines, as: :accountable
```
Comment (monkeyci) [inline app/models/concerns/accountable.rb]: Specify a :dependent option.<br>Specify an :inverse_of option.

File: app/models/concerns/accountable.rb [hunk 3]
```diff
@@ -0,0 +1,8 @@
+
+module Accountable
```
Comment (monkeyci) [inline app/models/concerns/accountable.rb]: Unnecessary blank line at the beginning of the source.

File: spec/commands/transfer_spec.rb [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+        amount: -amount_traded,
+      ).count
+    ).to eq(1)
+  end
+
+  it "creates ledger line for account_to" do
+    expect(
+      LedgerLine.where(
+        ledger_account: account_to,
+        amount: amount_traded,
```
Comment (monkeyci) [inline spec/commands/transfer_spec.rb]: Avoid comma after the last parameter of a method call.

File: spec/commands/transfer_spec.rb [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+      countable: invoice_product,
+      date: invoice_product.created_at
+    )
+  end
+
+  it "creates ledger line for account_from" do
+    expect(
+      LedgerLine.where(
+        ledger_account: account_from,
+        amount: -amount_traded,
```
Comment (monkeyci) [inline spec/commands/transfer_spec.rb]: Avoid comma after the last parameter of a method call.

**Comment by mf222** [comment]
[Contabilizar pagos y retiros](https://trello.com/c/4Qx6eIHv/56-contabilizar-pagos-y-retiros)

**Pull request merged**